### PR TITLE
add(phoenix): Move to Phoenix 1.3

### PR DESCRIPTION
**Pull request by kirill-kruchkov**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,28 +1,34 @@
...
-/cover/
-
-# The directory Mix downloads your dependencies sources to.
-/deps/
+# Generated on crash by the VM
+erl_crash.dump
 
-# Where 3rd-party dependencies like ExDoc output generated docs.
-/doc/
+# Generated on crash by NPM
```
Comment (MKenin) [inline .gitignore]: I hope we won't use node for anything, so you can safely remove this section.

File: config/dev.exs [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+# Do not include metadata nor timestamps in development logs
+config :logger, :console, format: "[$level] $message\n"
+
+# Set a higher stacktrace during development. Avoid configuring such
+# in production as building large stacktraces may be expensive.
+config :phoenix, :stacktrace_depth, 20
+
+# Configure your database
+config :elatrix,
```
Comment (MKenin) [inline config/dev.exs]: We do not need postgres config section either

**Review by MKenin — APPROVED** [review]
It's fine as it is

**Comment by kirill-kruchkov** [comment]
Fixed, merging

**Pull request merged**